datahub-classify/src/datahub_classify/infotype_helper.py

```diff
@@ -25,7 +28,15 @@
 )
 
 logger = logging.getLogger(__name__)
-nlp_english = load_spacy("en_core_web_sm")
+spacy_model_name = "en_core_web_sm"
+try:
+    nlp_english = spacy.load(spacy_model_name)
+except OSError:
+    logger.info("Downloading language model for the spaCy")
+    from spacy.cli import download
+
+    download(spacy_model_name)
```

Comment:
the spacy-download (https://pypi.org/project/spacy-download/) package was added in another PR - will this cause conflicts?

Reply:
We have used the spacy library itself to download the model. So, we don't require any other library dependency to download the spacy model. We have removed spacy-download from setup.py, so, the above code will not cause a conflict

Reply:
That sounds good